go/libkb/context.go

```diff
@@ -392,6 +392,8 @@ func (m MetaContext) switchUserNewConfig(u keybase1.UID, n NormalizedUsername, s
 			return err
 		}
 	}
+	// TODO: do i need to do anything particular for snooze_updates when user
```

Comment:
do i need to do anything particular for snooze_updates when user switching?

Reply:
I believe that updater settings is per-install and not per-user but @thebearjew should confirm

Reply:
Actually don't know the answer to this off the top of my head. I'll look into it.